centos7.6/opencv.cyg

```
@@ -0,0 +1,57 @@
+##############################################################################
+# maali cygnet file for OpenCV 
+# maali -t opencv 
+##############################################################################
+
+read -r -d '' MAALI_MODULE_WHATIS << EOF
+OpenCV (Open Source Computer Vision Library) is an open source computer vision and machine learning software library. 
+EOF
+
+# specify which compilers we want to build the tool with
+MAALI_TOOL_CPU_TARGET="$MAALI_DEFAULT_CPU_TARGET"
+MAALI_TOOL_COMPILERS="gcc/7.2.0 gcc/8.3.0"
+MAALI_TOOL_CUDA_COMPILERS="$MAALI_DEFAULT_CUDA_COMPILERS"
+MAALI_TOOL_PREREQ="python/3.6.3 ffmpeg/4.2.3 numpy/1.13.3 cuda/10.2"
```

cuda/10.2 shouldn't be here because is in MAALI_TOOL_CUDA_COMPILERS

---

Good catch, I forgot it there. Also, I would like the command to fail if `-n` is not specified.